### DISCUSSION TITLE
connector build failing

### DISCUSSION THREAD

**Discussion by Akshay-a-j** [eclipse-edc/Connector#3182] [Q&A] [answered]
When I try to build the connector using "java -jar build/libs/connector-0.1.1-SNAPSHOT.jar" command, it throws no main manifest attribute error. 
Despite adding the main class to the build.gradle file, the project still fails to recognize it, resulting in the inability to build the connector. Is there another way to debug this error?

**Comment by paullatzelsperger** [comment, marked as answer]
that is not a runnable jar. I would strongly suggest taking a look at the [Samples](https://github.com/eclipse-edc/Samples) and familiarise yourself with EDC in general and the code base in particular. EDC is not a runnable app, it is a framework.

**Comment by anton-ub** [comment]
In this tutorial video they are running it https://www.youtube.com/watch?v=O-YmJJp0nk8&list=PLw-f_YoTxWJVVPkuj1vDb6tLPM2_Cm1hR
But project structure is a bit different now. I have the same question: how to run the connector now?
